### PR TITLE
Create stale.yml

### PR DESCRIPTION
**Pull request by rthadur**
Adding this file to nag issue

### CODE REVIEW

**Comment by alexeagle** [comment]
FYI I tried this as well and it was rejected in #14172

**Comment by meteorcloudy** [comment]
@rthadur is actually from the gTech team I mentioned in the other PR.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
```
Comment (meteorcloudy) [inline .github/stale.yml:2]: I feel this time is a bit too short? It's possible people are just on vacation for two week and they intend to get back to the issue afterwards.

**Review by meteorcloudy — COMMENTED** [review]
@rthadur Can you check if there is anything we can improve by learning from https://github.com/bazelbuild/bazel/pull/14172?

**Comment by meteorcloudy** [comment]
@alexeagle Can you help review this to see if the settings are ideal?

File: .github/stale.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+markComment: >
+  This issue has been automatically marked as stale because it has no
+  recent activity. It will be closed if no further activity occurs. Thank you.
+# Comment to post when removing the stale label. Set to `false` to disable
+unmarkComment: false
+closeComment: >
+  Closing as stale. Please reopen if you'd like to work on this further.
+limitPerRun: 30
+# Limit to only `issues` or `pulls`
+only: issues
```
Comment (meisterT) [inline .github/stale.yml:18]: nit: in the comments above you talk about PRs as well, so perhaps update them?

File: .github/stale.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an Issue or Pull Request becomes stale
+daysUntilStale: 7
+# Number of days of inactivity before a stale Issue or Pull Request is closed
+daysUntilClose: 7
+# Only issues or pull requests with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels:
+ - stat:awaiting response
```
Comment (meisterT) [inline .github/stale.yml:7]: did you mean this one? https://github.com/bazelbuild/bazel/labels/awaiting-user-response

File: .github/stale.yml (hunk 1 above)
Comment (rthadur) [reply .github/stale.yml:2]: Thank you @meteorcloudy , If we do not receive reply from user in 7 days we apply stale label and wait another 7 days to close the issue(Total 14 days) , Here is an example [issue](https://github.com/tensorflow/tensorflow/issues/57633).
We can definitely customize this value , please suggest based on your requirement.

File: .github/stale.yml (hunk 2 above)
Comment (rthadur) [reply .github/stale.yml:18]: question , do you want this bot for only issues or you want to extend it to PRs ?

File: .github/stale.yml (hunk 3 above)
Comment (rthadur) [reply .github/stale.yml]: Done

File: .github/stale.yml (hunk 1 above)
Comment (fmeum) [reply .github/stale.yml:2]: I personally would prefer a longer time frame, perhaps 14 + 14 days instead of 7 + 7?

File: .github/stale.yml (hunk 2 above)
Comment (brentleyjones) [reply .github/stale.yml:18]: Since it's predicate on the label, I think PRs are valid as well.

File: .github/stale.yml (hunk 1 above)
Comment (meteorcloudy) [reply .github/stale.yml:2]: 14 + 14 sounds good to me, let's start with this

File: .github/stale.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+daysUntilClose: 14
+# Only issues with all of these labels are checked if stale. Defaults to `[]` (disabled)
+onlyLabels:
+ - awaiting-user-response
+# Comment to post when marking as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has no
+  recent activity. It will be closed if no further activity occurs. Thank you.
+# Comment to post when removing the stale label. Set to `false` to disable
+unmarkComment: false
```
Comment (meisterT) [inline .github/stale.yml:13]: what does this do?

Comment (rthadur) [reply .github/stale.yml:13]: if user responds  with in 14 days period , this will help remove the stale label .

**Review by meisterT — APPROVED** [review]

File: .github/stale.yml [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+closeComment: >
+  Closing as stale. Please reopen if you'd like to work on this further.
+limitPerRun: 30
```
Comment (meteorcloudy) [inline .github/stale.yml:16]: What does `limitPerRun` mean? Can you put a comment here?

File: .github/stale.yml (hunk 4 above)
Comment (meteorcloudy) [reply .github/stale.yml:13]: Do we need to create the "stale" label for this? We currently don't have one.

File: .github/stale.yml (hunk 5 above)
Comment (alexeagle) [reply .github/stale.yml:16]: I did a lot of helpful commenting in my version of this file https://github.com/bazelbuild/bazel/pull/14172/files

@rthadur I'd feel better as a ghosted contributor if you'd at least look through that and borrow the useful parts... 👻

Comment (rthadur) [reply .github/stale.yml:16]: Haha , sure let me check , thank you

File: .github/stale.yml (hunk 4 above)
Comment (rthadur) [reply .github/stale.yml:13]: Yes we need to create one stale label, thank you

Comment (meteorcloudy) [reply .github/stale.yml:13]: Created

**Comment by rthadur** [comment]
Can we merge this PR ?

File: .github/stale.yml (hunk 2 above)
Comment (brentleyjones) [reply .github/stale.yml:21]: Seems we are still limiting it only to issues?

**Review by meteorcloudy — APPROVED** [review]

**Pull request closed without merging**